src/session.rs

```diff
@@ -164,3 +163,163 @@ pub async fn clean_db(db: &SessionDBConn) -> Result<(), Error> {
     .await?;
     Ok(())
 }
+#[cfg(not(test))]
```

Comment:
This seems to imply that we test different code than is actually run in production, which I would not be a fan of.